src/psmoveservice/Device/Enumerator/VirtualControllerEnumerator.cpp

```diff
@@ -0,0 +1,56 @@
+// -- includes -----
+#include "VirtualControllerEnumerator.h"
+#include "ServerUtility.h"
+#include "assert.h"
+#include "hidapi.h"
+#include "string.h"
+
+//-- Statics
+int VirtualControllerEnumerator::virtual_controller_count= 0;
+
+// -- VirtualControllerDeviceEnumerator -----
+VirtualControllerEnumerator::VirtualControllerEnumerator()
+    : DeviceEnumerator(CommonDeviceState::VirtualController)
+{
+	m_deviceType= CommonDeviceState::VirtualController;
+    m_device_index= 0;
+
+    m_current_device_identifier= "VirtualController_0";
```

Comment:
Looks like I had an incorrect format specifier in VirtualControllerEnumerator::next()

https://github.com/cboulay/PSMoveService/pull/430/files/d600c7e10954cf81914de4df63e1b8afd8f0cabc#diff-a997644b7b062f7f3b123ada114ce214R50